packages/astro/src/core/render/core.ts

```diff
@@ -108,9 +108,10 @@ export type RenderPage = {
 	renderContext: RenderContext;
 	env: Environment;
 	apiContext?: APIContext;
+	isCompressHTMl?: boolean
 };
 
-export async function renderPage({ mod, renderContext, env, apiContext }: RenderPage) {
+export async function renderPage({ mod, renderContext, env, apiContext, isCompressHTMl = false }: RenderPage) {
```

Comment:
```suggestion
export async function renderPage({ mod, renderContext, env, apiContext, isCompressHTML = false }: RenderPage) {
```

The name isn't spelt right here (ant it's consumers).